mkdocs.yml

```diff
@@ -27,4 +27,6 @@ nav:
   - Pastas & similar:
       - Gnocchi with Pumpkin: "main-courses/pastas/gnocchi_with_pumpkin.md"
   - Breakfast:
-      - Porridge: "breakfast/porridge.md"
\ No newline at end of file
+      - Porridge: "breakfast/porridge.md"
+  - Beverage:
+      - Mate: "beverage/mate.md"
```

Comment:
I've created a preview and unfortunately it doesn't render:

![image](https://user-images.githubusercontent.com/28093618/140757289-da1d3623-7953-404d-8ae5-c57461f2513a.png)

Tip: the path to the file in mkdocs.yml is `beverage/mate.md`, but the recipe is under `Beverages/Mate.md ` --> they have to be the same (capital letters count)